ext/gmp/tests/bug80560.phpt

```diff
@@ -55,7 +56,6 @@ $functions2 = [
     'gmp_or',
     'gmp_xor',
     'gmp_hamdist',
-    'gmp_nextprime',
 ];
```

Comment:
Incorrectly classified functions were previously hidden because ArgumentCountError is a TypeError.